AzurePullRequests.Shared/appsettings.json

```diff
@@ -2,9 +2,9 @@
   "ProjectName": "eShopOnWeb",
   "Repository": "", //Guid for RepositoryId
```

Comment:
Commenting on a changed file for for context!

![images (1)](https://github.com/user-attachments/assets/5cfde595-aef6-4497-9afa-3e4ec85a3bb1)